docs/package.json

```diff
@@ -14,9 +14,10 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "^3.2.0",
-    "@docusaurus/preset-classic": "^3.2.0",
-    "@docusaurus/theme-mermaid": "^3.2.0",
+    "@docusaurus/core": "^3.4.0",
```

Comment:
do we need to update these as part of the PR?

Reply:
As I added a new dependency (`@docusaurus/plugin-client-redirects`) defaulting to `3.4.0` and all docusaurus versions must match, I figured it made sense to just update all of them.

Reply:
if it's related to PR changes we can keep it